keyboards/gh60/keymaps/emiilsd/keymap.c

```diff
@@ -0,0 +1,39 @@
+#include QMK_KEYBOARD_H
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /*
+    ISO HHKB layout:
+        * 2u Backspace
+        * ISO Enter
+        * split left Shift
+        * split right Shift
+        * Caps as Control
+        * 1u/1.5u/7u/1.5u//1u bottom row
+  */
+
+  /* 0: qwerty */
+  LAYOUT(
+    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_NO,
+    KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_BSLS, KC_ENT,
+    KC_LSFT, KC_GRV,  KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, F(0),    KC_RSFT,
```

Comment:
You could actually use `MO(1)` here, instead of the function, and `fn_actions` block.

Reply:
Thanks, didn't know that. Just used the default keymapping and edited to suit my layout. Would you say it would be better to use MO(1) then?

Reply:
Yeah, the "function" stuff is a holdover from TMK, I believe. There are "aliases" for all of the function code, that should make this "easier to read" in the keymap.

Reply:
@drashna Please do check newest newest commit